common/sku_customization.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ex
+
+# setup systemd service
+cat <<EOF >/usr/sbin/setup_topo.sh
+#!/bin/bash
+
+metadata_endpoint="http://169.254.169.254/metadata/instance?api-version=2019-06-04"
+vmSize=\$(curl -H Metadata:true \$metadata_endpoint \
```

Comment:
maybe parse vmSize directly using jq?

Reply:
I avoided jq as our images doesn't have the utility pre-installed. We do install it from the pipeline though. Do you suggest we install it in the scripts here?

Reply:
yeah, good point. It will be good to have jq also installed through the main scripts.